cmd/packages.go

```diff
@@ -255,7 +255,7 @@ func packagesExecWorker(userInput string) <-chan error {
 
 		src, cleanup, err := source.New(userInput, appConfig.Registry.ToOptions())
 		if err != nil {
-			errs <- fmt.Errorf("failed to determine image source: %w", err)
+			errs <- fmt.Errorf("analysis failed: %w", err)
```

Comment:
I think the error message still reads ok ok when it says `failed to determine image source` - It helps determine if the error came on sourcing the image VS processing it's packages `analysis failed` is a little vague and could belong to any part of the program

Reply:
I think you're right, we can make this message a bit more specific. "Failed to determine image source" is wrong, though, since this code is no longer used just to determine an _image_ source.

Reply:
`Failed to parse image source: %s, %w` Where `%v` is the `userInput`